android-test/build.gradle

```diff
@@ -31,6 +32,12 @@ android {
     versionCode 1
     versionName "1.0"
     testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+
+    // AndroidJUnitRunner wasn't finding tests in multidex artifacts when running on Android 4.0.3.
+    // Work around by adding all Okio classes to the keep list. That way they'll be in the main
+    // .dx file where TestRequestBuilder will find them.
+    multiDexEnabled true
+    multiDexKeepProguard file('multidex-config.pro')
```

Comment:
Getting here was a long and frustrating debug session.

Reply:
(We don’t exceed any 65K limits, but coreLibraryDesugaring needs multidex)

Reply:
I’m treating Android 4.0.3 as the lowest possible version any reasonable application would target. I’d like, for example, to have current versions of Okio 2.x working underneath OkHttp 3.12.x for example.